docs/redoc.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>WhimApp TSP API documentation</title>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:300,400,700|Roboto:300,400,700" rel="stylesheet" />
+
+    <link rel="icon" type="image/png" href="assets/openapi-icon.png" />
+    <link rel="apple-touch-icon" type="image/png" href="assets/openapi-icon.png" />
+    <link rel="canonical" href="https://github.com/maasglobal/maas-tsp-api" />
+    <meta name="author" content="MaaS Global Oy" />
+    <meta name="theme-color" content="#0095eb" />
+    <meta
+      name="description"
+      content="Documentation for WhimApp TSP API which enables transport service providers to integrate their services into WhimApp."
+    />
+    <meta property="twitter:card" content="summary_large_image" />
+    <meta property="twitter:site" content="@maas_global" />
+    <meta property="og:site_name" content="WhimApp TSP API" />
+    <meta property="og:url" content="https://github.com/maasglobal/maas-tsp-api" />
+    <meta property="og:title" content="WhimApp TSP API" />
+    <meta
+      property="og:description"
+      content="Documentation for WhimApp TSP API which enables transport service providers to integrate their services into WhimApp."
+    />
+    <meta property="og:locale" content="en-GB" />
+    <meta property="og:updated_time" content="2019-11-07T20:58:10.862Z" />
+    <meta property="og:image" content="assets/openapi-icon.png" />
+    <meta property="og:type" content="article" />
+
+    <!--
+    ReDoc doesn't change outer page styles
+    -->
+    <style>
+      body {
+        margin: 0;
+        padding: 0;
+        overflow-y: scroll;
+      }
+
+      .github-corner {
+        position: absolute;
+        display: block;
+        top: 0;
+        border: 0;
+        right: 0;
+        z-index: 9999;
+      }
+
+      .github-corner:hover .octo-arm {
+        animation: octocat-wave 560ms ease-in-out;
+      }
+
+      @keyframes octocat-wave {
+        0%,
+        100% {
+          transform: rotate(0);
+        }
+
+        20%,
+        60% {
+          transform: rotate(-25deg);
+        }
+
+        40%,
+        80% {
+          transform: rotate(10deg);
+        }
+      }
+
+      @media (max-width: 500px) {
+        .github-corner:hover .octo-arm {
+          animation: none;
+        }
+
+        .github-corner .octo-arm {
+          animation: octocat-wave 560ms ease-in-out;
+        }
+      }
+    </style>
+  </head>
+
+  <body>
+    <a href="https://github.com/maasglobal/maas-tsp-api" class="github-corner" aria-label="View source on Github">
+      <svg width="80" height="80" viewBox="0 0 250 250" style="fill:#202020; color:#fff" aria-hidden="true">
```

Comment:
Can this SVG be kept in a separate file, to make it easier to review the HTML?

Reply:
Done